ts.js

```diff
@@ -113,6 +113,7 @@
       ],
       '@typescript-eslint/type-annotation-spacing': 'error',
       '@typescript-eslint/unified-signatures': 'error',
+      'consistent-return': 'off',
```

Comment:
should we move these non `@typescript` cases above all the typescript configs?  it feels weird to have this one, and the one below, in amongst so many other ts-specific ones.

Reply:
disregard, i see the pattern now.

Reply:
I chose to leave this here because it kinda-sorta "goes with" `@typescript-eslint/explicit-function-return-type` below.